### PR TITLE
Added a lot

### PR DESCRIPTION
**Pull request by legomannetje**
Changed in this PR:
- AHK
  In this PR is a AHK example file + config for this example
  
- README.md
  The readme includes the changes, and links to CAD files for enclosures
  
- Windows page change scripts
  In python and in powershell scripts, for everyone something
  And for the python scipts an install guide and pip requirements file
  
- a FAQ
  A personal lookup guide to quickly help others with common issues.

### CODE REVIEW

**Review by koriwi — COMMENTED** [review]
autohotkey/config.bin 
is it correct that this file is empty? If yes, why do we need it?

File: faq/README.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+See the BOM with the PCB you are using
+
+### Where can I find resources
+You can find resources in the various github repo's, or you can ask us in the [discord](https://discord.gg/sEt2Rrd)
+
+### Where can I order the PCB
+We do have a good experience with [JLCPCB](https://jlcpcb.com) (Not affilated). You can also find some other company like EuroCurcuits
+
+### What do i need to order the PCB
+The GERBER-file zip that you can find in the hardware github repo
```
Comment (koriwi) [inline faq/README.md]: maybe add a direct link to the file here

File: faq/README.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+## Manufaturing
```
Comment (koriwi) [inline faq/README.md]: Manufacturing

File: faq/README.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+#### Through hole PCB (Adviced to starters in soldering)
+You will need:
+- PCB / Perfboard
+- Arduino Pro Micro (32u4, 16MHz, 5V)
+- 2x 4051 Multiplexers
+- 6x SSD1306 0.96" OLED
+- 6x 3x4x2mm SMD buttons
+- SPI-SDcard reader (Micro SD form)
+
+#### SMD varient (For advanced solderers)
```
Comment (koriwi) [inline faq/README.md]: variant

File: faq/README.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+### What do i need to order the PCB
+The GERBER-file zip that you can find in the hardware github repo
+
+## Manufaturing
+
+### Is there a specific order to solder the components in?
+Yes, we are currently making a guide on how to solder the components
+
+### Some pins are harder to solder
+Correct, the GND pins are harder to get to temperature. Have some patient, and let the solder flow. If it doesn't flow, the board might be bad on the long run.
```
Comment (koriwi) [inline faq/README.md]: patience

File: faq/README.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+(Insert pictures of good and bad soldering connections as example)
+
+### I did something wrong, and now?
+Try carefully to heat all the pins. What ever you do, **DO NOT FORCE IT, YOU WILL DAMAGE SOME PARTS**
+
+## Programming
+
+### Libraries are missing
+In case the Arduino IDE says you're missing some libraries, please install the SD-FAT and HID-Project libraries. How? [Read this guide](https://www.arduino.cc/en/guide/libraries)
```
Comment (koriwi) [inline faq/README.md]: SD-Fat needs 1.x
2.x won't work. I think it would be good to mention here.

File: faq/README.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+#### software issue
+Have you programmed it?
+
+#### Check the voltage
+The voltage between GND and VCC should be between 5V and 4V. At SCK and SDA it should be around 3V3. If one of these is not correct, you have some loose wire somewhere
+
+### The screens are white
+(Insert picture)
+#### Formatting SD card
+The SD card should be formatted mbt FAT32 (On windows, just FAT32)
```
Comment (koriwi) [inline faq/README.md]: mbt -> mbr

File: faq/README.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+#### Image cache
+Please play around with the settings.h image cache, this can give a lot of issues
```
Comment (koriwi) [inline faq/README.md]: 128 and 512 are good starting values

File: faq/README.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+Then its a not that common issue. Ask it in the discord!
+
+## How can i support this project?
+Currently there is not really  a way, but you can always give us a star on Github
+
+## Who are the contributors?
+The main contributors are:
+- Koriwi
+- AdamWelchUk
+- All of you the Freedeck users
```
Comment (koriwi) [inline faq/README.md]: Please add yourself here.
And maybe put youtube/first PCB behind adam in brackets :)

File: windows/auto_page_switch.py [hunk 9]
```diff
@@ -0,0 +1,40 @@
+
+import psutil
+import serial
+import time
+import threading, win32gui, win32process, psutil
+from win32gui import GetWindowText, GetForegroundWindow
+
+# Everything for the serial connection
+freedeckPort = "COM12" # or \dev\ttyACM0
```
Comment (koriwi) [inline windows/auto_page_switch.py]: linux uses normal slashes, so it would be /dev/ttyACM0

File: windows/page_list.txt [hunk 10]
```diff
@@ -0,0 +1,4 @@
+ts3client_linux,1
```
Comment (koriwi) [inline windows/page_list.txt]: ts3client_win64

**Review by koriwi — CHANGES_REQUESTED** [review]
autohotkey/config.bin 
is it correct that this file is empty? If yes, why do we need it?

Thanks for this pull request. Saving myself a lot of work. 
I really appreciate it!

File: faq/README.md (hunk 1 above)
Comment (legomannetje) [reply faq/README.md]: Then we also need to keep it up to date. But it has been added!

**Review by koriwi — APPROVED** [review]

**Pull request merged**